vhdbuilder/packer/components.json

```diff
@@ -316,7 +302,7 @@
         "v1.7.0"
       ],
       "multiArchVersions": [
-        "v1.9.0"
```

Comment:
could you verify this change? seems https://github.com/Azure/AgentBaker/pull/1395/files did not touch it

Reply:
yes, you're right. azurefile-csi:v1.8.0 is the first version with 'multi-arch'. I will use 1.8.0 instead of 1.9.0.